src/App.vue

```diff
@@ -11,20 +11,9 @@ import axios from "axios";
 
 export default {
   name: "App",
-   created() {
-    var url = "./env/config.json";
-    axios
-      .get(url)
-      .then(response => {
-        axios.defaults.baseURL = `${response.data.base_api_url}/`;
-        this.configLoaded = true;
-      })
-      .catch(() => {});
-  },
-  data() {
-      return {
-        configLoaded: false,
```

Comment:
The whole data object should not be removed because configLoaded needs to be initialized first if you want to use it